lib/common/error_private.c

```diff
@@ -38,6 +38,7 @@ const char* ERR_getErrorString(ERR_enum code)
     case PREFIX(tableLog_tooLarge): return "tableLog requires too much memory : unsupported";
     case PREFIX(maxSymbolValue_tooLarge): return "Unsupported max Symbol Value : too large";
     case PREFIX(maxSymbolValue_tooSmall): return "Specified maxSymbolValue is too small";
+    case PREFIX(stabilityCondition_notRespected): return "pledged buffer stability condition is not respected";
```

Comment:
`ZSTD_checkBufferStability()` is currently using `srcBuffer_wrong` and `dstBuffer_wrong`. If you're going to add this code, can you please switch them over too?

Reply:
Yes, good idea